source/includes/api/_dataset.md

```diff
@@ -491,239 +432,220 @@ curl -X POST https://api.resourcewatch.org/v1/dataset/find-by-ids \
-env                 | The environment you want the dataset to be accessible on. Defaults to production |  Text | `staging`, `production` or `preproduction` | No
+- In order to be able to create a dataset, you need to be [authenticated](#authentication).
+- Depending on your user account's role, you may have permission to create a dataset but not delete it afterwards.
+- All data uploaded to the RW API will be publicly visible and available to other users. 
 
+The first thing to consider when creating a dataset is where your data currently is - this will determine the [Dataset provider](#dataset-providers) you will need to use and, with it, a set of things you need to take into account - we'll cover each provider in detail shortly. When building your request to the RW API to create your dataset, you will need to take into account both the general details that follow, plus the details for the `provider` you are using. Be sure to review both sections when creating your datasets, to avoid any pitfalls.
 
-Once created, datasets have an auto-generated and unique slug and id that allows the user to get, create, update, query or clone that dataset. The dataset slug and the id cannot be modified.
+Creating a dataset is done using a POST request and passing the relevant data as body files. The supported body fields are as defined on the [dataset reference](#dataset-reference) section, but the minimum field list you must specify for all datasets is:
 
-When a dataset is created, the `status` is set to "pending" by default, meaning the dataset is not ready for usage just yet. Once the creation process is finished, the status will change to "saved", meaning you can now access the dataset and its data. This process is asynchronous and its duration will depend on the provider of the dataset as well as the amount of data it contains. You should use [check your dataset](#how-to-get-a-dataset-by-id) manually to see when the `status` is updated.
 
-In case the dataset creation process fails, the `status` value will be set to `failed`, and the `errorMessage` field will have a short description of what went wrong.
+- name
+- application
+- connectorType
+- provider 
 
-When creating a dataset, depending on the provider you are using, there are slight differences that must be reflected in your request:
+Additionally, depending on the data provider you will be using, other fields may be required - we'll cover those in detail in each provider's specific documentation.
 
-### Carto datasets
+A successful dataset creation request will return a 200 HTTP code, and the dataset details as stored on the RW API. PAy special attention to the `id` or the `slug`, as those will allow you to [access your dataset](#getting-a-dataset-by-id-or-slug) later.
```

Comment:
It would also be good to have an overview table on what you can do with datasets from different providers and where the difference are. Probably not in this document but rather in the dataset-concept section

Reply:
Added a new section on the "concept" part about this.